app/services/notification_service.rb

```diff
@@ -1,6 +1,13 @@
 class NotificationService
 
-  def self.notify(account:, project:, issue_id:, issue_comment_id: nil)
+  def self.notify(account_id:, project_id:, issue_id:, issue_comment_id: nil)
+    project = Project.find(project_id)
+    account = Account.find(account_id)
+
+    self.do_notify(account, project, issue_id, issue_comment_id)
+  end
+
+  def self.do_notify(account, project, issue_id, issue_comment_id)
```

Comment:
Why did you break this out into a separate method?

Reply:
because I had to pass the `ids` to the method. Find the object and then call the original hardcoded method. I could rewrite this to be in one single method though.

Reply:
If you could, that would be perfect, thanks!